src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -33,6 +33,10 @@ def load_arguments(self, _):
     with self.argument_context('spring-cloud create') as c:
         c.argument('location', arg_type=get_location_type(
             self.cli_ctx), validator=validate_location)
+        c.argument('reserved_cidr_range', help='A CIDR notation IP range from which to host underlying infrastructure. Can be 1 unused /14 IP range(or as large as available), or 3 unused /16 IP ranges. Required when deploying into a Virtual Network.', validator=validate_vnet)
```

Comment:
@yucwan Could you pls review the user interface part first?

Reply:
Propose to change this reserved_cidr_range to:

c.argument('reserved_address_prefixes', help=' Space-separated list of address prefixes in CIDR format. The IP range is reserved to host underlying Azure Spring Cloud infrastructure, which can be 1 unused /14 IP range(or as large as available), or 3 unused /16 IP ranges. Required when deploying into a Virtual Network.', validator=validate_vnet)

And please provide at least 2 examples for how to use it.

Reply:
I know the VNet uses address-prefixes/address-prefix in its CLI and API. But I would prefer the CIDR here because
1. We define the API's property as `serviceCidr`, for consistency, we can still use CIDR here.
1. Address-prefix is more like a self defined term by vnet team, not a concept used world-wide like CIDR. Even the resources like `route-table` under network team doesn't align with the address-prefix term.

Reply:
c.argument('reserved_cidr_range', help=' ??-separated list of IP address ranges in CIDR format. The IP ranges are reserved to host underlying Azure Spring Cloud infrastructure, which can be 1 unused /14 IP range(or as large as available), or 3 unused /16 IP ranges. Required when deploying into a Virtual Network.', validator=validate_vnet)